File: SampleSetup/Output/Swift/Model/AbstractModelWithId.swift
struct AbstractModelWithId: Codable {
    let Id: String

    let InsertUserName: String
    let InsertDate: String?

    let UpdateUserName: String
    let UpdateDate: String?

    let RemoveIs: Bool
    let RemoveUserName: String
    let RemoveDate: String?
}

File: SampleSetup/Output/PHP/SampleProject/Protocol/Model/AbstractModelWithId.php

<?php
namespace Virta\SampleProject\Protocol\Model;

class AbstractModelWithId {
    /** @var string */ public $Id;

    /** @var string */ public $InsertUserName;
    /** @var string|null */ public $InsertDate;

    /** @var string */ public $UpdateUserName;
    /** @var string|null */ public $UpdateDate;

    /** @var bool */ public $RemoveIs;
    /** @var string */ public $RemoveUserName;
    /** @var string|null */ public $RemoveDate;
}
